containers/kubernetes/cockpit.conf.tmpl

```diff
@@ -6,6 +6,7 @@ Shell = /shell/simple.html
 {{else}}
 Shell = /shell/stub.html
 {{if .is_openshift }}
+ProtocolHeader = X-Forwarded-Proto
```

Comment:
It's a de facto standard, commonly used in proxy configurations. But there are others in use.

If we wanted to be really flexible we could support configuring the expected value as such as the microsoft ``Front-End-Https: on``` or the new  ```Forwarded: ``` header that I've not yet seen in the wild.